haskell-lsp-types/haskell-lsp-types.cabal

```diff
@@ -58,6 +58,7 @@ library
                      , scientific
                      , text
                      , unordered-containers
+                     , either >= 5.0.1.1
```

Comment:
Nitpicking here, if we’re only using those two helper functions from either would it be worthwhile to just write them ourselves as a `where` clause? Just to avoid adding more dependencies

Reply:
Thanks! Will do.

Reply:
Done.